### DISCUSSION TITLE
Using react-query without react

### DISCUSSION THREAD

**Discussion by geekox86** [TanStack/query#3680] [Q&A]
Greetings,

Thank you for this great library.
I am trying to use it with xstate.
I do not need the react hooks.
When I tried to use QueryClient and QueryObserver directly, I got an error for missing react-dom.

What can I do to achieve this goal?

Many thanks.

**Comment by TkDodo** [comment]
Can you show this in a codesandbox reproduction?

**Reply by geekox86** [reply]
I found the solution to import react-query/core.

**Reply by tomitrescak** [reply]
Do you have an example of this? There is nothing in react dom

**Reply by ejabu** [reply]
@geekox86 could you please share the example of it ?
